.github/workflows/ci-workflow.yml

```diff
@@ -0,0 +1,51 @@
+name: test-suite
+on: 
+  push:
+    branches:
+    - master
```

Comment:
I think it's quite important we also do this other branches. And can we do these Actions on external PRs too?

Reply:
Yes this is fixed now for all branches and pull requests: 

"When you create a pull request from a forked repository to the base repository, GitHub sends the pull_request event to the base repository and no pull request events occur on the forked repository." : so this will trigger the CI workflow at the base repository